keyboards/meetlab/kafkasplit/config.h

```diff
@@ -0,0 +1,34 @@
+/* Copyright 2022 LXF-YZP(yuezp)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* Double tap reset button to enter bootloader */
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_LED GP25
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 500U
+
+
+#define SERIAL_USART_FULL_DUPLEX
+#define SERIAL_USART_TX_PIN GP0
+#define SERIAL_USART_RX_PIN GP1
+
+#define I2C1_SCL_PIN  GP27
+#define I2C1_SDA_PIN  GP26
+#define I2C_DRIVER I2CD1
```

Comment:
It won't negatively impact things, but it might be worth setting the i2c driver to run at 400kHz, rather than the default 100kHz. 
```suggestion
#define I2C_DRIVER I2CD1
#define I2C1_CLOCK_SPEED 400000
```

This has the advantage of running faster, and should update the screen faster, and allow it to move on, faster.

Reply:
> It won't negatively impact things, but it might be worth setting the i2c driver to run at 400kHz, rather than the default 100kHz.
> 
> This has the advantage of running faster, and should update the screen faster, and allow it to move on, faster.

Thank you very much for your advice